.github/workflows/test.yml

```diff
@@ -47,7 +47,50 @@ jobs:
         make docker-test-integration
     - name: Upload cAdvisor log file
       uses: actions/upload-artifact@v1
-      if: failure()
+      if: success() || failure()
       with:
-        name: cadvisor.log
-        path: ${{ github.workspace }}/go/src/github.com/google/cadvisor/cadvisor.log
+        name: cadvisor-${{ matrix.distro }}-${{ matrix.arch }}-${{ matrix.go-versions}}-${{ github.run_number }}-${{ github.run_id }}.log
```

Comment:
There is no `matrix.distro` and `matrix.arch` in `test-integration` job.
This leads to e.g.
```
Uploading artifact 'cadvisor---1.20-1899-4410275830.log' from '/home/runner/work/cadvisor/cadvisor/cadvisor.log' for run #1793
```
Please provide arch variable. Could you use platform instead of distro?

Reply:
Why do you want to keep logs when everything seems to be working?